### PR TITLE
Cross post new dask dataframe post to dask blog

### PR DESCRIPTION
**Pull request by scharlottej13**
Another very dask-forward post that seems like a good fit for the Dask blog!

@phofl mind reviewing?

cc @jacobtomlinson

### CODE REVIEW

File: _posts/2024-05-30-dask-is-fast.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+tags: [dask, query optimizer, dataframe]
+theme: twitter
+canonical_url: https://docs.coiled.io/blog/dask-dataframe-is-fast.html
+---
+
+{% include JB/setup %}
+
+<meta content="Dask DataFrame is faster and more reliable, especially for TBs of data. This is due to engineering improvements like adding a query optimizer, integrating with Apache Arrow, and more efficient data shuffling." name="description" />
+
+_This work was engineered and supported by [Coiled](https://coiled.io/?utm_source=dask-blog&utm_medium=dask-expr) and [NVIDIA](https://www.nvidia.com/). Thanks to [Patrick Hoefler](https://github.com/phofl) and [Rick Zamora](https://github.com/rjzamora), in particular. Original version of this post appears on [docs.coiled.io](https://docs.coiled.io/blog/dask-dataframe-is-fast.html?utm_source=dask-blog&utm_medium=dask-is-fast)_
```
Comment (scharlottej13) [inline _posts/2024-05-30-dask-is-fast.md:14]: @phofl is this accurate?

**Review by jacobtomlinson — APPROVED** [review]
Looks good to me, thanks @scharlottej13

File: _posts/2024-05-30-dask-is-fast.md (hunk 1 above)
Comment (phofl) [reply _posts/2024-05-30-dask-is-fast.md:14]: Hm the list should probably be longer for this post since it includes shuffles and arrow strings.

**Comment by phofl** [comment]
Post looks good to me

File: _posts/2024-05-30-dask-is-fast.md (hunk 1 above)
Comment (scharlottej13) [reply _posts/2024-05-30-dask-is-fast.md:14]: Thanks @phofl! Anyone in particular you have in mind?

**Comment by jacobtomlinson** [comment]
Thanks all!

**Comment by scharlottej13** [comment]
Thanks @jacobtomlinson for the review!

**Pull request merged**